### ISSUE TITLE
Errors in console when running tests

### ISSUE THREAD

**Issue by eugef** [vitalets/angular-xeditable#595]
Hi @ckosloski I've noticed that browser console is full of errors, while allt the tests are green.

Here are examples of those errors:
```
test.js:9 Uncaught ReferenceError: inject is not defined
    at angular.scenario.Describe.<anonymous> (test.js:9)
    at angular.scenario.Describe.<anonymous> (angular-scenario.js:43828)
    at angular.scenario.Describe.describe (angular-scenario.js:43359)
    at angular.scenario.Runner.describe (angular-scenario.js:43824)
    at angular-scenario.js:11332
    at test.js:1
(anonymous) @ test.js:9
(anonymous) @ angular-scenario.js:43828
angular.scenario.Describe.describe @ angular-scenario.js:43359
angular.scenario.Runner.describe @ angular-scenario.js:43824
(anonymous) @ angular-scenario.js:11332
(anonymous) @ test.js:1
```
```
VM34012 angular.js:14195Error: [$sce:unsafe] Attempting to use an unsafe value in a safe context.
http://errors.angularjs.org/1.5.10/$sce/unsafe
    at http://127.0.0.1:8000/bower_components/angular/angular.js:68:12
    at htmlSanitizer (http://127.0.0.1:8000/bower_components/angular/angular.js:18702:13)
    at getTrusted (http://127.0.0.1:8000/bower_components/angular/angular.js:18871:16)
    at Object.sce.(anonymous function) [as getTrustedHtml] (http://127.0.0.1:8000/bower_components/angular/angular.js:19550:16)
    at ngBindHtmlWatchAction (http://127.0.0.1:8000/bower_components/angular/angular.js:25517:29)
    at Scope.$digest (http://127.0.0.1:8000/bower_components/angular/angular.js:17832:23)
    at Scope.$apply (http://127.0.0.1:8000/bower_components/angular/angular.js:18098:24)
    at bootstrapApply (http://127.0.0.1:8000/bower_components/angular/angular.js:1808:15)
    at Object.invoke (http://127.0.0.1:8000/bower_components/angular/angular.js:4767:19)
    at doBootstrap (http://127.0.0.1:8000/bower_components/angular/angular.js:1806:14)
```

Could you please check and possibly fix it?

**Comment by ckosloski** [comment]
I will take a look.

**Comment by ckosloski** [comment]
@eugef when I run the doc tests in Safari I see a lot of these:

> [Error] Failed to load resource: the server responded with a status of 404 (Not Found) (ng-tags-input.css.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 (Not Found) (ng-tags-input.bootstrap.css.map, line 0)

**Comment by eugef** [comment]
Did you run `npm install` to install dependencies?

**Comment by ckosloski** [comment]
Yes, the error only appears to occur when running the tests in safari

**Comment by eugef** [comment]
Errors, you have sent are not critical, just map files are missing for minifed css.

I've fixed the error related to `Uncaught ReferenceError: inject is not defined` https://github.com/vitalets/angular-xeditable/commit/1024d7ca9d02d488c82932c293353dcd0237eb37. 
Turns out that this test  was broken and didn't run for a long time, but `angular-scenario` didn't report that.

So the only error that is left is `[$sce:unsafe] Attempting to use an unsafe value in a safe context` which occurs if you run `dev-tests`
